src/odemis/driver/smarpod.py

```diff
@@ -350,29 +349,34 @@ def __init__(self, name, role, locator, axes=None, **kwargs):
         # will take care of executing axis move asynchronously
         self._executor = CancellableThreadPoolExecutor(1)  # one task at a time
 
-        # Check referencing, and reference if not referenced
+        axes_ref = {a: False for a, i in self.axes.items()}
+        # VA dict str(axis) -> bool
         self.referenced = model.VigilantAttribute(axes_ref, readonly=True)
         referenced = c_int()
         self.core.Smarpod_IsReferenced(self._id, byref(referenced))
         if not referenced.value:
-            logging.debug("SmartPod is not referenced. The device will not function until referencing occurs.")
+            logging.debug("SmarPod is not referenced. The device will not function until referencing occurs.")
```

Comment:
Move this log in an `else` block of the `if`. Also, make it a little higher level: "warning".